pkg/restclient/common.go

```diff
@@ -0,0 +1,73 @@
+package restclient
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+const baseURL = "http://localhost:24007"
+
+// RESTClient represents Glusterd2 REST Client
+type RESTClient struct {
```

Comment:
Just `Client` is a good name for this. The doc comment anyway specifies what sort of a client this is.